sdks/python/apache_beam/testing/load_tests/pardo_test.py

```diff
@@ -132,22 +137,38 @@
 @unittest.skipIf(not load_test_enabled, 'Enabled only for phrase triggering.')
 class ParDoTest(LoadTest):
   def setUp(self):
-    self.output = self.pipeline.get_option('output')
-    self.iterations = self.pipeline.get_option('number_of_counter_operations')
+    super(ParDoTest, self).setUp()
+    self._apply_filter()
+    self.iterations = self._get_option_or_default('iterations')
+    self.nb_of_counter = self._get_option_or_default('number_of_counters')
```

Comment:
I would go with full name of `number`.

Reply:
`nb` is a quite popular abbreviation of `number`. What's more, shorter variable names are better, so I'd rather keep `nb` in this case.

Reply:
It doesn't seem to be that popular abbreviation across Beam (sometimes `new_block` was called `nb` as well). But ok until it's just local variable for this test.

Reply:
I agree with Kasia - `number_of_counters` is easier to digest. As a non python dev and a person that didn't know this code, I had to think for a little while what does `nb` mean. `number`says it explicitly and costs nothing